channels/us.m3u

```diff
@@ -3445,75 +3445,75 @@ https://dai.google.com/linear/hls/event/Xu-I6qGiSTKeu6glzS4BtQ/master.m3u8
+https://rpn1.bozztv.com/36bay2/gusa-tvssportsbureau/tracks-v1a1/index-1623403141-now.m3u8
 #EXTINF:-1 tvg-id="TVSTallyHo.us" tvg-name="TVS Tally Ho" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/t0ly2m8.jpg" group-title="Classic",TVS Tally Ho (360p)
-https://nrpus.bozztv.com/36bay2/gusa-tvstallyho/index.m3u8
+https://rpn1.bozztv.com/36bay2/gusa-tvstallyho/tracks-v1a1/mono.m3u8
 #EXTINF:-1 tvg-id="TVSTavern.us" tvg-name="TVS Tavern" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/5UjqDht.jpg" group-title="Classic",TVS Tavern (720p)
-https://nrpus.bozztv.com/36bay2/gusa-tavern/index.m3u8
-#EXTINF:-1 tvg-id="TVSTelevisionNetwork.us" tvg-name="TVS Television Network" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/pXOyNEy.png" group-title="",TVS Television Network (490p)
-http://nrpus.bozztv.com/36bay2/gusa-tvstn/index.m3u8
+https://rpn1.bozztv.com/36bay2/gusa-tavern/tracks-v1a1/index-1623403445-now.m3u8
+#EXTINF:-1 tvg-id="TVSTelevisionNetwork.us" tvg-name="TVS Television Network" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/pXOyNEy.png" group-title="General",TVS Television Network (490p)
+https://rpn1.bozztv.com/36bay2/gusa-tvstn/tracks-v1a1/index-1623403948-now.m3u8
 #EXTINF:-1 tvg-id="TVSTodayHomeEntertainmentNetwork.us" tvg-name="TVS Today Home Entertainment Network" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/pXOyNEy.png" group-title="",TVS Today Home Entertainment Network (720p)
 https://nrpus.bozztv.com/36bay2/gusa-TVSTodayHomeEntertainment/index.m3u8
-#EXTINF:-1 tvg-id="TVSTodayHomeEntertainmentNetwork.us" tvg-name="TVS Today Home Entertainment Network" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/pXOyNEy.png" group-title="",TVS Today Home Entertainment Network
-https://nrpus.bozztv.com/36bay2/gusa-TVSTodayHomeEntertainment/tracks-v1a1/index.m3u8
-#EXTINF:-1 tvg-id="TVSTravelNetwork.us" tvg-name="TVS Travel Network" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/Bgqy4fL.jpg" group-title="Travel",TVS Travel Network (720p)
-https://nrpus.bozztv.com/36bay2/gusa-tvstravel/index.m3u8
-#EXTINF:-1 tvg-id="TVSTurbo.us" tvg-name="TVS Turbo" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/RMewTGr.jpg" group-title="Classic",TVS Turbo (360p)
-https://nrpus.bozztv.com/36bay2/gusa-tvsturbo/index.m3u8
+#EXTINF:-1 tvg-id="TVSTodayHomeEntertainmentNetwork.us" tvg-name="TVS Today Home Entertainment Network" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/pXOyNEy.png" group-title="General",TVS Today Home Entertainment Network
+https://rpn1.bozztv.com/36bay2/gusa-TVSTodayHomeEntertainment/tracks-v1a1/index-1623403745-now.m3u8
+#EXTINF:-1 tvg-id="TVSTravelandFood" tvg-name="TVS Travel & Food" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/Bgqy4fL.jpg" group-title="Travel",TVS Travel & Food Network (720p)
```

Comment:
Incorrect tvg-id

Reply:
you are correct. Old link was dead. I cannot find tvg-id for the new channel. can cancel the pull request. I will pull that channel

Reply:
I mean you just didn't place .us in the end, no need to close the request

Reply:
just did another pull. i will update that channel with the next pull. still going through the list